apps/platform/src/app/(main)/page.tsx

```diff
@@ -138,28 +94,37 @@ export default function Index(): JSX.Element {
           localStorage.getItem('currentWorkspace') ?? '{}'
         ) as Workspace)
       : (JSON.parse(`{}`) as Workspace)
+  
+  useEffect( () => {
 
-  useEffect(() => {
-    Projects.getProjectsbyWorkspaceID(currentWorkspace.id)
-      .then((data: ProjectWithoutKeys[] | [] | undefined) => {
+    console.log("Current workspace slug: ", currentWorkspace.slug);
+
+    const projectController = new ProjectController(
+          process.env.NEXT_PUBLIC_BACKEND_URL
+    )
+    
+    projectController.getAllProjects({workspaceSlug: currentWorkspace.slug}, {})
+      .then((data: GetAllProjectsResponse) => {
         if (data) {
```

Comment:
Thsi isnt the correct pattern. you will get back `data`, `success` and `error`. You would need to check if success is true and then extract it from data. You should use object destructuring in line 107

Reply:
> Thsi isnt the correct pattern. you will get back `data`, `success` and `error`. You would need to check if success is true and then extract it from data. You should use object destructuring in line 107

Cool, I'll do this.